var/spack/repos/builtin/packages/py-dask/package.py

```diff
@@ -130,7 +135,8 @@ class PyDask(PythonPackage):
     depends_on('py-toolz@0.8.2:',       type=('build', 'run'), when='@2.13.0: +delayed')
 
     # Support for YAML configuration files
-    depends_on('py-pyyaml',             type=('build', 'run'), when='+yaml')
```

Comment:
If there are no objections to what I wrote regarding the conflict statements, we don't need to change anything here.